src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -47,62 +40,6 @@ public HostController(WebScriptHostManager scriptHostManager, WebHostSettings we
             _authorizationService = authorizationService;
         }
 
-        [HttpPost]
-        [Route("admin/functions/{name}")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
-        [RequiresRunningHost]
-        [EnableDebugMode]
-        public IActionResult Invoke(string name, [FromBody] FunctionInvocation invocation)
-        {
-            if (invocation == null)
-            {
-                return BadRequest();
-            }
-
-            FunctionDescriptor function = _scriptHostManager.Instance.GetFunctionOrNull(name);
-            if (function == null)
-            {
-                return NotFound();
-            }
-
-            ParameterDescriptor inputParameter = function.Parameters.First(p => p.IsTrigger);
-            Dictionary<string, object> arguments = new Dictionary<string, object>()
-            {
-                { inputParameter.Name, invocation.Input }
-            };
-            Task.Run(() => _scriptHostManager.Instance.CallAsync(function.Name, arguments));
-
-            return Accepted();
-        }
-
-        [HttpGet]
-        [Route("admin/functions/{name}/status")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevel)]
-        [RequiresRunningHost]
-        public IActionResult GetFunctionStatus(string name)
-        {
-            FunctionStatus status = new FunctionStatus();
-            Collection<string> functionErrors = null;
-
-            // first see if the function has any errors
-            if (_scriptHostManager.Instance.FunctionErrors.TryGetValue(name, out functionErrors))
-            {
-                status.Errors = functionErrors;
-            }
-            else
-            {
-                // if we don't have any errors registered, make sure the function exists
-                // before returning empty errors
-                FunctionDescriptor function = _scriptHostManager.Instance.Functions.FirstOrDefault(p => p.Name.ToLowerInvariant() == name.ToLowerInvariant());
-                if (function == null)
-                {
-                    return NotFound();
-                }
-            }
-
-            return Ok(status);
-        }
-
```

Comment:
These two routes were accidentally duplicated in another commit. Since these new tests use these APIs externally, they hit 'ambiguous route' exceptions.

We should spin up some end-to-end REST API tests to make sure we catch these issues before committing in the future.